R/fd_observations.R

```diff
@@ -0,0 +1,115 @@
+#' Number of observations in the FoRTE Dataset
+#'
+#' @note No. of observations for many, but not all datasets
+#'
+#' #' @return A `data.frame` or `tibble` of the observation tally.
+#' Call \code{\link{fd_metadata}} for field metadata.
+#' @export
+#'
+#' @examples
+#' fd_observations()
+fd_observations <- function() {
```

Comment:
It didn't occur to me that this might be a package function; was expecting simply code in the RMarkdown. Hmm.